### PR TITLE
Add new severities to spec

### PR DESCRIPTION
**Pull request by nporteschaikin**
info, minor, major, critical, blocker

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ An `issue` represents a single instance of a real or potential code problem, det
 * `location` -- **Required**. A `Location` object representing the place in the source code where the issue was discovered.
 * `trace` -- **Optional.** A `Trace` object representing other interesting source code locations related to this issue.
 * `remediation_points` -- **Optional**. An integer indicating a rough estimate of how long it would take to resolve the reported issue.
-* `severity` -- **Optional**. A `Severity` string (`info`, `normal`, or `critical`) describing the potential impact of the issue found.
+* `severity` -- **Optional**. A `Severity` string (`info`, `minor`, `major`, `critical`, or `blocker`) describing the potential impact of the issue found.
```
Comment (GordonDiggs) [inline SPEC.md]: I'm assuming I missed the conversation on this, but why is it ok to remove "normal"?

Comment (nporteschaikin) [inline SPEC.md]: Our app will return `nil` for `severity` in this case, which we handle gracefully.

Comment (GordonDiggs) [inline SPEC.md]: Cool - sounds good

**Comment by wfleming** [comment]
@nporteschaikin This seems like it should necessitate a version bump of the spec in `VERSION` to `0.3.0`.

File: SPEC.md (hunk 1 above)
Comment (wfleming) [inline SPEC.md]: I think there might have been some confusion around this: relevant documents return `nil` when no severity is set, but if any engine explicitly emitted "normal" severity, then I'm fairly sure that's what we'll return in app & elsewhere.

Brakeman also, today, emits issues with "normal" severity, so brakeman is no longer spec-compliant as of this change.

Comment (nporteschaikin) [inline SPEC.md]: @wfleming This is correct/was discussed IRL; my comment above was unclear.

**Pull request merged**